source/docs/articles/wordpress/wordpress-and-pantheon-varnish-cache.md

```diff
@@ -0,0 +1,62 @@
+---
+title: WordPress and Pantheon's Varnish Servers
+description: Detailed information on understanding the interactions of WordPress Pantheon's Varnish edge caching servers.
```

Comment:
+1 for @ari-gold's suggestion to change title to **Programatically Using Pantheon Cache Functions** also suggest changing the file name to match